app/org/sagebionetworks/bridge/dynamodb/DynamoParticipantOptionsDao.java

```diff
@@ -118,8 +116,6 @@ public OptionLookup getOptionForAllStudyParticipants(StudyIdentifier studyIdenti
         scan.addFilterCondition("studyKey", condition);
         
         List<DynamoParticipantOptions> mappings = mapper.scan(DynamoParticipantOptions.class, scan);
-        List<FailedBatch> failures = mapper.batchDelete(mappings);
-        BridgeUtils.ifFailuresThrowException(failures);
```

Comment:
BTW this should already have been removed in a separate PR I had Eric review and submit immediately, because it was destructive.